test/jdk/sun/security/provider/acvp/Launcher.java

```diff
@@ -58,19 +66,33 @@
 /// [https://github.com/usnistgov/ACVP?tab=readme-ov-file#supported-algorithms].
 ///
 /// Example:
+///
+/// Run locally with ArtifactResolver
 /// ```
-/// jtreg -Dtest.acvp.provider=SunJCE \
-///       -Dtest.acvp.alg=ML-KEM \
-///       -Dtest.acvp.data=/path/to/json-files/ \
-///       -jdk:/path/to/jdk Launcher.java
+/// jtreg -Djdk.test.lib.artifacts.ACVP-Server=<path-to-archive-file>
 /// ```
+/// OR host the zip archive on artifactory server.
+///
+
 public class Launcher {
 
     private static final String ONLY_ALG
             = System.getProperty("test.acvp.alg");
 
     private static final Provider PROVIDER;
 
+    private static final String ACVP_BUNDLE_LOC = "jpg.tests.jdk";
+    private static final String ACVP_BUNDLE_NAME = "ACVP-Server";
+    private static final String ACVP_BUNDLE_VERSION = "1.1.0.38";
+    // Zip archive entry name, do not update to use File.separator
+    private static final String[] TEST_FILES = {
+            "gen-val/json-files/ML-DSA-keyGen-FIPS204/internalProjection.json",
+            "gen-val/json-files/ML-DSA-sigGen-FIPS204/internalProjection.json",
+            "gen-val/json-files/ML-DSA-sigVer-FIPS204/internalProjection.json",
+            "gen-val/json-files/ML-KEM-encapDecap-FIPS203/internalProjection.json",
+            "gen-val/json-files/ML-KEM-keyGen-FIPS203/internalProjection.json"
+    };
+
```

Comment:
One more comment, since we only run on `TEST_FILES`, we should require all of them to succeed. I suggest a real exception to be thrown when `invalidTest` or `unsupportedTest` is not zero.